manimpango/cmanimpango.pyx

```diff
@@ -264,6 +264,12 @@ def text2svg(
     return file_name
 
 class MarkupUtils:
+    @staticmethod
+    def validate(text: str) -> bool:
+       cdef GError** res = NULL
+       text_bytes = text.encode("utf-8")
+       return pango_parse_markup(text_bytes, len(text_bytes), 0, NULL, NULL, NULL, res)
```

Comment:
I think `res` is NULLABLE, so if we aren't using it then it can be NULL. Also, `len(text_bytes)` isn't required. Instead it can be `-1` and Pango calculates the length.
```suggestion
       return pango_parse_markup(text_bytes, -1, 0, NULL, NULL, NULL, NULL)
```

Reply:
You are right, it actually works even if `res` is `NULL`. The docs said the function would return `false` if `error` was set, so I'd better give a real pointer. For the length, I suggest keeping this. It does not cost us anything and automatic determination of length will fail if `\0` is in the string (no idea why anyone would want that in their string, but still). My experience tells me it is always better to indicate the length, if this can be done "for free".

Reply:
Looks like this is causing some compiler warnings on Windows.
See https://github.com/ManimCommunity/ManimPango/runs/1696088350?check_suite_focus=true#step:5:96
doing like this fixes it
```py
return pango_parse_markup(text_bytes, <int>len(text_bytes), 0, NULL, NULL, NULL, res)
```
I have done this in #12